Android/build_and_install.sh

```diff
@@ -1,4 +0,0 @@
-#!/bin/bash
```

Comment:
Let's update the RELEASING.md too to indicate that we don't need to run this anymore